### PR TITLE
feat(Tools): Update Tools + Hubs

### PR DESCRIPTION
**Pull request by manuelbarbas**

### CODE REVIEW

File: src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_csharp.mdx
Comment (TheGreatAxios) [inline src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_csharp.mdx:1]: I think we should add a note here that says this only works in C# on Mobile/Desktop at this time and not WebGL due to coroutines

File: src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_dart.mdx
Comment (TheGreatAxios) [inline src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_dart.mdx:1]: Should we just add skale.dart here to simplify?

File: src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_javascript.mdx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        let resultHash = nonceAddressXOR.xor(candidateHash);
+        let externalGas = divConstant.div(resultHash).toNumber();
+        if (externalGas >= gasAmount) {
+            break;
+        }
+    }
+    return candidate.toString();
+}
+
+exports.mineGasForTransaction = mineGasForTransaction
```
Comment (TheGreatAxios) [inline src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_javascript.mdx:175]: add ; at end of line

File: src/styles/overrides.css [hunk 2]
```diff
@@ -65,3 +66,7 @@ pre,
 .right-sidebar-panel:where(.astro-pb3aqygn) a:hover {
 	background-color: var(--sl-color-hover);
 }
+
+mjx-container svg {
+	display: inline !important;
+}
```
Comment (TheGreatAxios) [inline src/styles/overrides.css:72]: Remove with switch to MathML

File: astro.config.mjs [hunk 3]
```diff
@@ -2,9 +2,15 @@ import { defineConfig } from "astro/config";
 import starlight from "@astrojs/starlight";
 import react from "@astrojs/react";
 import astroExpressiveCode from "astro-expressive-code";
+import remarkMath from 'remark-math';
+import rehypeMathjax from 'rehype-mathjax';
```
Comment (TheGreatAxios) [inline astro.config.mjs:6]: Remove -> switch to MathML

File: astro.config.mjs [hunk 4]
```diff
@@ -2,9 +2,15 @@ import { defineConfig } from "astro/config";
...
 
 // https://astro.build/config
 export default defineConfig({
+	markdown: {
+		remarkPlugins: [remarkMath],
+		rehypePlugins: [rehypeMathjax],
+	  },
```
Comment (TheGreatAxios) [inline astro.config.mjs:13]: Remove -> switch to MathML

File: src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_python.mdx
Comment (TheGreatAxios) [inline src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_python.mdx:1]: If taken from my repo (remove temporarily as code does not work)

**Review by TheGreatAxios — CHANGES_REQUESTED** [review]

File: src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_dart.mdx
Comment (manuelbarbas) [reply src/content/docs/tools/skale/sfuel-distribution/pow_examples/pow_dart.mdx:1]: We can my idea was just to have all examples with the same structure. But, for now, I'll add the https://pub.dev/packages/skale there. If you prefer just the repo I'll change

File: src/content/docs/tools/skale/sfuel-distribution/contract-distribution.mdx
[the remark this answers is not in the document]
Comment (manuelbarbas) [reply src/content/docs/tools/skale/sfuel-distribution/contract-distribution.mdx:1]: Do you mean to have a function to give, for example, MINTER_ROLE and distribute sFUEL. Then have other function for mint for MINTER_ROLE access?

File: src/content/docs/tools/skale/sfuel-distribution/pow-distribution.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+1. Get a wallet to sign transactions - a random generated one can be used since PoW based transactions don't require signer to have sFUEL
+2. Create a transaction to call sFUEL distribution contract function
+3. Unique gasPrice function generated - based on: signer address, signer nonce and gas
+4. Send transaction with the unique GasPrice
+
+##### Available On-Chain sFUEL Faucets
+
+| Chain | Mainnet Address | Testnet Address | Function Signature |
+| --------- | --------- | --------- | --------- |
+| Europa | [0x2B267A3e49b351DEdac892400a530ABb2f899d23](https://elated-tan-skat.explorer.mainnet.skalenodes.com/address/0x2B267A3e49b351DEdac892400a530ABb2f899d23) |[0x366727B410fE55774C8b0B5b5A6E2d74199a088A](https://juicy-low-small-testnet.explorer.testnet.skalenodes.com/address/0x366727B410fE55774C8b0B5b5A6E2d74199a088A) | 0x6a627842 |
```
[the remark this answers is not in the document]
Comment (manuelbarbas) [reply src/content/docs/tools/skale/sfuel-distribution/pow-distribution.mdx:48]: What does it mean to tab something? (sorry)

**Review by TheGreatAxios — APPROVED** [review]

**Pull request merged**